src/transformStateWithClones.js

```diff
@@ -6,8 +6,37 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
   // write code here
+  const array = [];
+  let stateCopy = { ...state };
+  let pushedCopy = {};
+
+  for (const object of actions) {
+    switch (object.type) {
+      case 'addProperties':
+        Object.assign(stateCopy, object.extraData);
+        break;
+
+      case 'removeProperties':
+        object.keysToRemove.forEach(Element => {
+          delete stateCopy[Element];
+        });
```

Comment:
You always want to start those type of things in lowercase (camelCase). Btw maybe more descriptive name like `key`?
```suggestion
        object.keysToRemove.forEach(element => {
          delete stateCopy[element];
        });
```